### PR TITLE
Correct documentation for `chunksize=True`

### PR DESCRIPTION
**Pull request by LeonLuttenberger**
### Feature or Bugfix
- Documentation

### Detail
- Correct documentation for `chunksize=True`

### Relates
- #2086

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: GitHubCodeBuild8756EF16-4rfo0GHQ0u9a
* Commit ID: 3b7fad2d87dfa9fe1019d4ed7517a6e59a0630c6
* Result: FAILED
* [Build Logs](https://gqgcj9s2gc.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=e8ee4274-05aa-4260-96de-88d76cb84a20%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: GitHubCodeBuild8756EF16-4rfo0GHQ0u9a
* Commit ID: 3b7fad2d87dfa9fe1019d4ed7517a6e59a0630c6
* Result: SUCCEEDED
* [Build Logs](https://gqgcj9s2gc.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=abb51580-ba52-4583-b584-e33de6dadeb7%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: awswrangler/s3/_read_parquet.py [hunk 1]
```diff
@@ -830,10 +832,12 @@ def read_parquet_table(
...
     There are two batching strategies on awswrangler:
 
-    - If **chunked=True**, a new DataFrame will be returned for each file in your path/dataset.
+    - If **chunked=True**, depending on the size of the data, one or more data frames will be
+      returned per each file in the path/dataset.
+      Unlike **chunked=INTEGER**, rows from different files will not be mixed in the resulting data frames.
 
     - If **chunked=INTEGER**, awswrangler will paginate through files slicing and concatenating
-      to return DataFrames with the number of row igual the received INTEGER.
+      to return DataFrames with the number of rows egual the received INTEGER.
```
Comment (malachi-constant) [inline awswrangler/s3/_read_parquet.py:840]: equal*

File: awswrangler/s3/_read_parquet.py [hunk 2]
```diff
@@ -600,9 +600,11 @@ def read_parquet(
 
     There are two batching strategies on awswrangler:
 
-    - If **chunked=True**, a new DataFrame will be returned for each file in your path/dataset.
+    - If **chunked=True**, depending on the size of the data, one or more data frames will be
+      returned per each file in the path/dataset.
+      Unlike **chunked=INTEGER**, rows from different files will not be mixed in the resulting data frames.
 
-    - If **chunked=INTEGER**, awswrangler will iterate on the data by number of rows igual the received INTEGER.
+    - If **chunked=INTEGER**, awswrangler will iterate on the data by number of rows egual the received INTEGER.
```
Comment (malachi-constant) [inline awswrangler/s3/_read_parquet.py:607]: equal*

File: awswrangler/s3/_read_parquet.py [hunk 3]
```diff
@@ -652,7 +654,7 @@ def read_parquet(
     chunked : Union[int, bool]
         If passed will split the data in a Iterable of DataFrames (Memory friendly).
         If `True` awswrangler iterates on the data by files in the most efficient way without guarantee of chunksize.
-        If an `INTEGER` is passed awswrangler will iterate on the data by number of rows igual the received INTEGER.
+        If an `INTEGER` is passed awswrangler will iterate on the data by number of rows egual the received INTEGER.
```
Comment (malachi-constant) [inline awswrangler/s3/_read_parquet.py:657]: equal*

**Review by malachi-constant — APPROVED** [review]
Looks good, just saw one last typo

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: GitHubCodeBuild8756EF16-4rfo0GHQ0u9a
* Commit ID: 14cf6aeb0fe0f834936cd7030597be9d4b8707f1
* Result: SUCCEEDED
* [Build Logs](https://gqgcj9s2gc.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=38e3ef4e-065e-4e80-8d9e-9fe3a71be132%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by kukushking — APPROVED** [review]
Thanks!

**Pull request merged**